src/api/graphhopper.d.ts

```diff
@@ -1,5 +1,6 @@
 import { LineString } from 'geojson'
 
+// min lat, min lng, max lat, max lng
```

Comment:
I think it is actually min lng, min lat, max lng, max lat, for example world = [-180, -90, 180, 90], but yes, adding such a comment might help :)

Reply:
oh, strange. I thought I did my research properly :/

Reply:
Uh, looks like I had an inconsistent state in my mind ;)

https://github.com/graphhopper/graphhopper-maps/compare/master...test_current_location#diff-516d68fe9c8c9fc0d5cc075d0ba6348d77cee435c4b9eb0bad4c844d9af215b6R22